tools.py

```diff
@@ -439,6 +440,9 @@ def filter_dataset(input_path,
 		seed):
 	dataset = transcripts.load(input_path)
 
+	if cer:
+		assert cer < 1.0, 'CER should be set in number less than 1.0'
```

Comment:
-> `assert not cer or 0 <= cer < 1.0, 'CER should be unset or in range [0.0, 1)'`

Reply:
done